Update angular to 1.2.10

julesbou opened the pull request.

julesbou committed:
updated angular to 1.2.7 (with ui.router)

julesbou committed:
updated angular to 1.2.7 (with ui.router)

dmitryevseev commented on bower.json:
```diff
@@ -8,10 +8,12 @@
+    "angular-resource": "latest",
+    "angular-ui-router": "latest",
+    "angular-mocks": "latest",
+    "angular-mocks": "latest",
```
duplicate

julesbou commented on bower.json:
```diff
@@ -3,15 +3,15 @@
-    "angular-mocks": "~1.0.7",
-    "angular-scenario": "~1.0.7",
+    "angular": "~1.2.8",
+    "angular-ui-router": "~0.2.8",
```
@dmitryevseev when doing a `bower install` there's some files missing for "angular-ui-router". Until this problem is fixed, this PR cannot be merged of course :)

dmitryevseev replied:
I've discussed with Iwein and Tiziano already idea about disposing deps to be checked into the repo. Just having a strict versions numbers in bower.json will be enough - going to do that.

julesbou commented:
@dmitryevseev finally updated to 1.2.10 instead of 1.2.7

dmitryevseev committed:
improve(ui.router): Gets rid of routes.js - state should be defined on module level.

dmitryevseev committed:
improve(ui.router): Gets rid of routes.js - state should be defined on module level.

dmitryevseev committed:
improve(gulp): Applies latest updates to gulpfile:
 - fix of gulp requirejs
 - adds error handlers to gulp tasks
 - disables csso minification and uses native sass instead

dmitryevseev committed:
Merge remote-tracking branch 'origin/feature/angular-update' into feature/angular-update

dmitryevseev committed:
fix(gulp requirejs): uses it right

dmitryevseev committed:
style(requirejs config): disposes obsolete angular mocks link in config-require.js

dmitryevseev committed:
fix(jasmine matchers): disposes jasmine matchers as these are not working with latest karma-jasmine
